nuget.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This can potentially impact tests as nuget.config are automatically imported.

Reply:
You mean the test projects in src\Assets pull in nuget.config? Should we add an empty nuget.config to that directory then? Alternatively, I think that setting `RestoreSources` property causes restore to ignore nuget.config, so perhaps the test projects should set that property?

---
In reply to: [184184799](https://github.com/dotnet/sdk/pull/2178#discussion_r184184799) [](ancestors = 184184799)

Reply:
I think we are already creating a nuget.config, but we should make sure that is the case, so that we don't start taking an accidental dependency on the root nuget.config getting auto-imported.

@dsplaisted, do you know if we are creating one for all tests, or just for the perf tests?

Reply:
As of #2157 a NuGet.config is generated for all tests (it goes in the root test execution folder)